test/common-algorithms/dynamic-programming/sequential/knapsack_recursive.chpl

```diff
@@ -0,0 +1,36 @@
+proc max(a:int, b:int): int{
```

Comment:
Use [built-in max](http://chapel.cray.com/docs/master/modules/internal/UtilMisc_forDocs.html#UtilMisc_forDocs.max)